### PR TITLE
Add Election to PersonPost

### PR DESCRIPTION
**Pull request by symroe**
This is a quick way to fix the fact that posts should have more than one election.  This will allow for filtering of people based on (election,post).

There is still a problem with posts and elections, but that will be addressed later.

### CODE REVIEW

File: wcivf/apps/people/managers.py [hunk 1]
```diff
@@ -63,20 +63,23 @@ def update_or_create_from_ynr(self, person):
...
+                        slug=membership['election']['id'])
                     elections.append(election)
 
                 if membership['post']:
                     post, _ = Post.objects.update_or_create(
                         ynr_id=membership['post']['id'],
                         label=membership['post']['label'],
                     )
+                    if election:
+                        post.election = election
```
Comment (tfgg) [inline wcivf/apps/people/managers.py]: Why do posts still have an election attribute?

Comment (symroe) [inline wcivf/apps/people/managers.py]: Sorry, this is confusing.  The election here is only being used later in this call…in `PersonPost.objects.update_or_create`.  This is only to pass the object through to that method.

Comment (symroe) [inline wcivf/apps/people/managers.py]: I mean, it's setting the post's election too – removing that will come in a future PR

Comment (tfgg) [inline wcivf/apps/people/managers.py]: That is quite confusing. Can the field be properly killed off and the election object be passed through properly?

Comment (symroe) [inline wcivf/apps/people/managers.py]: That creates quite a lot more work right now, and I'd rather just get this fix deployed so we're not showing wrong candidates, and then work on doing it properly when I'm more with it / can pair with someone…

Comment (tfgg) [inline wcivf/apps/people/managers.py]: Ok sure, go ahead then.

**Pull request merged**